docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,574 @@
+# Text
+
+There are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.
+
+- The [TextField](#textField) is to edit a single line text.
+
+- The [TextEditor](#textEditor) is to edit multi-line text.
+
+<a name="textLabel"></a>
+## TextLabel
+
+The `TextLabel` is a class that displays a short text string. `Tizen.NUI.BaseComponents` namespace contains the class.
+
+The `TextLabel` class is lightweight, non-editable, and do not respond to user input. Text labels support multiple languages and scripts including right-to-left scripts such as Arabic. For more information on how to display a text using a text label, see [NUI Hello World Tutorial](../../get-started/nui/quickstart.md).
+
+**Figure: Text label positioned to top left**
+
+![Text label example, positioned to top left](media/TextLabelTopLeft.png)
+
+<a name="create"></a>
+### Create Text Label

```

Comment:
```
### Create a TextLabel
```

Reply:
@mijin85cho 
I have seen other pages and I didn't feel "a" makes any difference. Rather we avoid using articles if the meaning is clear. 
https://docs.tizen.org/application/dotnet/guides/wcircularui/quickstart#create-project
https://docs.tizen.org/application/dotnet/guides/wcircularui/quickstart_tizenxamlapptemplate#create-project

Reply:
I think @surbhig12 shared you the issue on https://github.com/Samsung/tizen-docs/pull/1036, which contains the updated guideline for that.

https://github.com/Samsung/tizen-docs/pull/1036#pullrequestreview-366949552

![image](https://user-images.githubusercontent.com/9958374/77631449-ee76f000-6f8f-11ea-996f-eeb8031fdb59.png)

So this comment was suggested like below before.
https://github.com/Samsung/tizen-docs/pull/1009#discussion_r377385079
![image](https://user-images.githubusercontent.com/9958374/77631533-09e1fb00-6f90-11ea-87ed-cdfb3ac03851.png)

Reply:
@mijin85cho, as I can see PR 1036 is merged. We decided to remove articles, wherever the heading is clear for example in this case, Create TextLabel but retain "your" in headings (again scenario based).

However, in this case, I think the correct suggestion would be 
## Create TextLabel and not ## Create Text Label

If you are referring to a general text label, then I suggest it be ## Create a Text Label


Please let me know your views, so that we can finalize and communicate the same to all members.

Reply:
Hello, @surbhig12 

Thanks for the comment.
I gathered below the latest suggested headings from SRI-B during the review. (Except the page title.)


\## TextLabel
\### Create a TextLabel
\### Set Font of TextLabel
\### Set Font Styles of TextLabel
\### Align Text in a TextLabel
\### Use Decorations for Text
\### Use Markup Styling to Style Text
\## TextField
\### TextField Events
\### Create a TextField
\### Align Text of a TextField
\### TextField Properties
\### Use Decorations for TextField
\## TextEditor
\### TextEditor Events
\### Create a TextEditor
\### TextEditor Properties

Accepting your comment, and fixing some issues, how about below?
Would you please check them and give your opinion?

\## TextLabel
\### Create TextLabel
\### Set Font of TextLabel
\### Set Font Styles of TextLabel
\### Align Text in TextLabel
\### Use Decorations for TextLabel
\### Use Markup Styling to Style TextLabel
\## TextField
\### TextField Events
\### Create TextField
\### Align Text in TextField
\### TextField Properties
\### Use Decorations for TextField
\## TextEditor
\### TextEditor Events
\### Create TextEditor
\### TextEditor Properties

Reply:
@mijin85cho, thank you for accepting. I agree with your suggestions. All are fine.

Reply:
Thanks for the reply, @surbhig12.

@JoogabYun For this line, use below.

```
### Create TextLabel
```